### PR TITLE
Add notes regarding bsc#173158 - CONFIG_MODULE_SIG

### PR DESCRIPTION
**Pull request by lkocman**
Current WIP, this is an important change that needs to be visible.

### CODE REVIEW

File: xml/release-notes.xml [hunk 1]
```diff
@@ -513,31 +513,23 @@
...
+  <sect2 xml:id="sec.driver.sig">
+   <!-/- bsc#173158 -/->
+   <title>CONFIG_MODULE_SIG=y</title>
    <para>
-    With the kernel shipped in &thisflavor; 15.0 GM, the Nouveau driver for
-    Nvidia graphics card may hang at reboot, shutdown, or during runtime power
-    management actions. This bug occurs primarily on system with hybrid
-    graphics, such as laptops that include integrated Intel graphics and a
-    discrete Nvidia graphics card.
+     This is an important change for all users of 3rd party drivers using secureboot.
```
Comment (fsundermeyer) [inline xml/release-notes.xml:524]: How about adding an example after "3rd party drivers"? E.g. "(for example NVidia or AMD graphics drivers from the respective manufacturer's repositories)"

File: xml/release-notes.xml [hunk 2]
```diff
@@ -513,31 +513,23 @@
...
+     The module signature check is on &thisflavor; 15.2.
+     Unsigned modules will be tainted and will result in a warning message.
+     Signed modules that fail to check will be blocked.
```
Comment (fsundermeyer) [inline xml/release-notes.xml:527]: Sorry, but I fail to understand what this actually means. If e.g. NVidia drivers for 15.2 are signed against 15.2 how can their check fail? And why would there be unsigned modules? 
And further, what was the previous behavior and how and why has it changed?

File: xml/release-notes.xml [hunk 3]
```diff
@@ -513,31 +513,23 @@
...
+     See more details at <link xlink:href="https://en.opensuse.org/SDB:NVIDIA_drivers"/>
```
Comment (fsundermeyer) [inline xml/release-notes.xml:528]: There is no further information on this topic at all. And if it affects all 3rd party drivers, why would that information be on a page dealing with NVidia graphhic drivers only?

File: xml/release-notes.xml [hunk 4]
```diff
@@ -513,31 +513,23 @@
...
+ <sect1 xml:id="sec.driver">
   <title>Drivers and Hardware</title>
   <para/> -->
 
-  <!-- <sect2 xml:id="sec.driver.nvidia-hybrid">
-   <!-/- boo#1094834 -/->
-   <title>Hang on Machines with Nvidia GPUs and Hybrid Graphics</title>
+  <sect2 xml:id="sec.driver.sig">
+   <!-/- bsc#173158 -/->
+   <title>CONFIG_MODULE_SIG=y</title>
```
Comment (fsundermeyer) [inline xml/release-notes.xml:522]: This heading does not say anything useful for people not being familiar with the technical details: I suggest something more speaking like "Secure Boot: 3rd Party drivers need to be properly signed".
The CONFIG_MODULE_SIG=y setting then should be mentioned  below (maybe together with an explanation on how to turn it off (and what that would imply).

File: xml/release-notes.xml (hunk 2 above)
Comment (lkocman) [reply xml/release-notes.xml:527]: Well the entire point is to use drivers signed by vendor or unsigned. If you try a some driver which seems like it's signed but it was altered (somebody tries to inject something). Then it's right thing to block it.

**Comment by lkocman** [comment]
Hopefully all addressed.

**Comment by lkocman** [comment]
Merging because of the deadline.

**Pull request merged**